src/content/zh-tw/_shared/udacity/ud811.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN" "http://www.w3.org/TR/REC-html40/loose.dtd">
```

Comment:
original does not contain the doctype, `html` or `body` tag. These files are included in to other files and should not include them.